apps/agreement/contracts/Agreement.sol

```diff
@@ -989,7 +993,16 @@ contract Agreement is ILockManager, IAgreement, AragonApp {
         _challenge.state = ChallengeState.Voided;
 
         (DisputableAragonApp disputable, CollateralRequirement storage requirement) = _getDisputableFor(_action);
-        _transferTo(requirement.token, _challenge.challenger, requirement.challengeAmount);
+        address challenger = _challenge.challenger;
+        _transferTo(requirement.token, challenger, requirement.challengeAmount);
+        // return half of the fees to submitter and the other half to challenger
+        uint256 challengerArbitratorFeeAmount = _challenge.challengerArbitratorFeeAmount;
+        ERC20 challengerArbitratorFeeToken = _challenge.challengerArbitratorFeeToken;
+        uint256 submitterPayBack = challengerArbitratorFeeAmount / 2;
+        // no need for Safemath because of previous computation
+        uint256 challengerPayBack = challengerArbitratorFeeAmount - submitterPayBack;
+        _transferTo(challengerArbitratorFeeToken, _action.submitter, submitterPayBack);
+        _transferTo(challengerArbitratorFeeToken, challenger, challengerPayBack);
```

Comment:
```suggestion
        // No need for Safemath because of previous computation
        uint256 challengerPayBack = challengerArbitratorFeeAmount - submitterPayBack;
        _transferTo(challengerArbitratorFeeToken, _action.submitter, submitterPayBack);
        _transferTo(challengerArbitratorFeeToken, challenger, challengerPayBack);
        
```